### DISCUSSION TITLE
Add option to notify only on completed workflows

### DISCUSSION THREAD

**Discussion by lpulley** [community/community#12057] [Actions]
Today, there is an option to disable emails for successful workflows.

![image](https://user-images.githubusercontent.com/7193187/155582870-ebac6bfd-9a6c-477c-96ec-a209820151f0.png)

However, there is no option to disable emails for canceled workflows. When a PR is updated and the `concurrency` key cancels the previous checks, the PR author receives an email that their workflow was canceled. This is completely useless to us, and super annoying.

There should be a toggle, or some other configuration, that allows us to receive emails for successes and failures, but not cancellations.

**Comment by ethomson** [comment]
This is good feedback, @lpulley, I appreciate it.  We'll take a look at improving this for workflows that are cancelled by concurrency.

**Reply by lpulley** [reply]
Thanks! For now, some of us are setting up personal email filters to trash the cancellation results, but it's not ideal.
